Modules/CaPTkCommon/src/SpacedMaskPoints.hpp

```diff
@@ -0,0 +1,104 @@
+#ifndef SPACEDMASKPOINTSHPP_H
```

Comment:
delete this file

Reply:
Thanks @dboun I was wondering what that file is...

Reply:
Done, will update this PR momentarily